news/2024/2024-04-17-osutaiko-world-cup-2024-semifinals-recap.md

```diff
@@ -0,0 +1,93 @@
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2119605829&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+There's South Korea. Former podium-placers. A team with several top 50 players and tournament veterans, well-rounded and fit to cover everything the mappools have to offer. Seeding 3rd with balanced placements across the board. Then there's the United Kingdom, the former podium-placers. A team with top 50 players and tournament veterans, well-rounded and fit to…
+
+Hey… Wait a minute… Aren't they basically the same? Looking at the stats, the only thing separating these guys' seedings is a rank sum difference of *two*. And you're telling me they're up against each other now? In the winners' bracket semifinals?! In a battle to secure the aforementioned podium placement?! (Localized entirely within your kitchen?!)
+Well you better strap in, because this was exactly the type of roller coaster match you're imagining.
+
+The United Kingdom won the roll and came swinging right out of the gate with [Masayoshi Iimori - NEWALONE (Extended Mix)](https://osu.ppy.sh/beatmapsets/2163092#taiko/4561817), grabbing a quick point thanks to their spectacular accuracy. South Korea countered with [MYUKKE. - Mach Roger](https://osu.ppy.sh/beatmapsets/2163105#taiko/4561831), thinking they could make use of their Hidden trump card, ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027/taiko), but nuh-uh. The UK grabbed that one too, and continued to do so with two more picks, making it 4–0. A gut-wrenching sight for the Koreans.
+
+The United Kingdom had proven themselves to be *just a cut above*, but when they went fishing in the Free Mod pool with [Sparxe - Switcheroo](https://osu.ppy.sh/beatmapsets/2163116#taiko/4561848), South Korea found the opening they'd been looking for, grabbing a breakpoint and bouncing back with one after another. Some time passed and all of a sudden we were here again, the first tiebreaker since the Round of 32.
+
+Since we've jumped a few rounds, adding mods to the tiebreaker map is now nothing short of lunacy. But for two of the top 3 seeds and some guaranteed prize money on the line, that didn't seem to matter. A lone Hidden from South Korea, and a Hidden and Hard Rock from the United Kingdom. [Tektheist - Pomegranate](https://osu.ppy.sh/beatmapsets/2163122#taiko/4561855) did all it could to make sure both mods had a horrible time. But again, that didn't seem to matter. The Korean No Mod players ended up taking the brunt of the misses which pushed the lead further and further towards the UK. And despite being put through the wringer for a full five minutes, the United Kingdom made sure it's coming home, together with an unfathomable Hard Rock full combo from ::{ flag=GB }:: [goheegy](https://osu.ppy.sh/users/8057655/taiko).
```

Comment:
```suggestion
Since we've jumped a few rounds, adding mods to the tiebreaker map is now nothing short of lunacy. But for two of the top 3 seeds and some guaranteed prize money on the line, that didn't seem to matter. A lone Hidden from South Korea, and a Hidden and Hard Rock from the United Kingdom. [Tektheist - Pomegranate](https://osu.ppy.sh/beatmapsets/2163122#taiko/4561855) did all it could to make sure both mods had a horrible time. But again, that didn't seem to matter.

The Korean No Mod players ended up taking the brunt of the misses which pushed the lead further and further towards the UK. Not letting up one bit despite being put through the wringer for a full five minutes on the tiebreaker, the United Kingdom made sure to take it home, together with an unfathomable Hard Rock full combo from ::{ flag=GB }:: [goheegy](https://osu.ppy.sh/users/8057655/taiko).
```